debug/src/debug.js

```diff
@@ -6,6 +6,23 @@ export function initDebug() {
 	/* eslint-disable no-console */
 	let oldBeforeDiff = options.diff;
 
+	options.checkParent = parentNode => {
+		if (!parentNode) {
+			throw new Error('Undefined parent passed to render(), this is the second argument.\nCheck if the element is available in the DOM/has the correct id.');
+		}
+		let isValid;
+		switch (parentNode.nodeType) {
+			case 1:
+			case 3:
+			case 8:
+			case 9:
+			case 11: isValid = true; break;
+			default: isValid = false;
+		}
+		if (!isValid) throw new Error('Expected a valid HTML node as a second argument to render.');
```

Comment:
I could imagine new users being confused about which `render` the message is referring to. Perhaps just amending a second line where we print the component name would help out.

```js
`Expected a valid HTML node as a second argument to render. Received ${parentDom} instead: render(<MyComponent />, ${parentDom});`
```

Not sure if that is easily doable.